client/src/App.js

```diff
@@ -6,7 +6,7 @@ import { makeStyles } from '@material-ui/core';
 import Navigation from './components/Navigation';
 import Auth from './components/auth';
 import Snackbar from './components/common/Snackbar';
-
+import Home from './components/home/Home';
```

Comment:
Rename the file to `index.js` and import as such:
```suggestion
import Home from './components/home';
```